packages/ndla-ui/src/Embed/ImageEmbed.tsx

```diff
@@ -10,23 +10,20 @@ import parse from "html-react-parser";
 import { MouseEventHandler, ReactNode, useMemo, useState } from "react";
 import { useTranslation } from "react-i18next";
 import styled from "@emotion/styled";
-import { colors, spacing, utils } from "@ndla/core";
+import { colors, spacing } from "@ndla/core";
 import { ExpandTwoArrows } from "@ndla/icons/action";
 import { ArrowCollapse, ChevronDown, ChevronUp } from "@ndla/icons/common";
 import { COPYRIGHTED } from "@ndla/licenses";
 import { ImageEmbedData, ImageMetaData } from "@ndla/types-embed";
 import EmbedErrorPlaceholder from "./EmbedErrorPlaceholder";
-import { CanonicalUrlFuncs, HeartButtonType, RenderContext } from "./types";
+import { RenderContext } from "./types";
 import { Figure, FigureType, figureActionIndicatorStyle } from "../Figure";
-import Image, { ImageLink } from "../Image";
+import Image from "../Image";
 import { EmbedByline } from "../LicenseByline";
 
 interface Props {
   embed: ImageMetaData;
   previewAlt?: boolean;
-  path?: string;
-  heartButton?: HeartButtonType;
-  canonicalUrl?: CanonicalUrlFuncs["image"];
```

Comment:
Skal vi ikke støtte canonical lenger? Er det sånn at bilder ikke skal være klikkbare lenger?

Reply:
Trur ikkje det. Lenker til bilder skal kun finnes i Regler for bruk. Og klikk på sidestilte bilder skal ekspanderes, slik som på aftenposten.no